token/program-2022/src/state.rs

```diff
@@ -303,7 +303,7 @@ impl GenericTokenAccount for Account {
     fn valid_account_data(account_data: &[u8]) -> bool {
         // Use spl_token::state::Account::valid_account_data once possible
         account_data.len() == Account::LEN && is_initialized_account(account_data)
-            || (account_data.len() >= Account::LEN
+            || (account_data.len() > Account::LEN
```

Comment:
I can't comment further down, but with this change, we can even simplify to:

```
ACCOUNTTYPE_ACCOUNT == *account_data[Account::LEN]
```

What do you think?

Reply:
Yeah, should be safe to change :+1: